### PR TITLE
API changes

### PR DESCRIPTION
**Pull request by badassorange**
**Features:**
Hide banner and enable CORS
Moved config to dir, added example and gitignore
Added: auth method without redirect, response content type json

**Other:**
Updated README.md
Updated postman collection

### CODE REVIEW

**Comment by badassorange** [comment]
Please, someone who is good at golang, give feedback)

**Comment by JackBekket** [comment]
@ChronosX88 check please, I guess it's your part

**Comment by JackBekket** [comment]
Just a tiny note - we suppose, that we will separate programmers and reqruiters somewhere at front page (and will show them their personal about and 'how to use' page)

So, when *programmer* try to auth - we want to get his profile data and check if he has *email* and have setted *'available for hire'* at settings. 

If he doesn't have these info - we would like to tell him, that he must set email and check hirable at settings.

I want to ask if we can make a separate auth method for user? 
I could make an extention to library and make one single function, which check requirements from profile and return bool flag to auth process if you want ^__^

File: cmd/spork-backend/main.go [hunk 1]
```diff
@@ -16,9 +16,13 @@ func main() {
 	//Set logger level
 	echoServ.Logger.SetLevel(log.DEBUG)
 
+	// Hide banner
+	echoServ.HideBanner = true
+
 	//Echo middleware init
 	echoServ.Use(middleware.Logger())
 	echoServ.Use(middleware.Recover())
+	echoServ.Use(middleware.CORS())
```
Comment (vrumg) [inline cmd/spork-backend/main.go:25]: В этом месте могут быть проблемы с префлайт запросами. Сталкивался у себя в проекте с тем, что нужно в явном виде задать AllowOrigins AllowMethods и AllowHeaders. Мы вроде не передаем никаких токенов в заголовках, AllowCredentials можно в false оставить. Заголовки он мне вообще явно футболил, пока я не передал те которые реально приходят. 
Вот пример:

s.Router.Use(middleware.CORSWithConfig(middleware.CORSConfig{
		AllowOrigins:     []string{"http://1", "https://2", "https://3"},
		AllowMethods:     []string{"OPTIONS", "GET", "PUT", "POST", "DELETE", "HEAD"},
		AllowHeaders:     []string{"Access-Control-Allow-Methods", "Access-Control-Request-Headers", "Access-Control-Request-Method", "Authorization", "Content-Type"},
		AllowCredentials: true,
	}))

**Pull request merged**